x-pack/plugins/maps/common/types.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface ImportResponse {
```

Comment:
I don't think we should use these types from file_upload. They are designed for the very generic endpoint needed to create unknown indices. For this use case, we know the index shape so the request/response should be tailored for this patterns. I would like to see the response sent to the client to be less elasticsearchy and more aligned to the API.